cypress/integration/test.spec.js

```diff
@@ -0,0 +1,6 @@
+/// <reference types="Cypress" />
```

Comment:
Hope this line represents the cypress default

Reply:
We have updated the code as per the review comments